src/pre-commit

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env node
+
+const childProcessExec = require('child_process').exec;
+const util = require('util');
+
+const exec = util.promisify(childProcessExec);
+
+if (!String.prototype.startsWith) {

```

Comment:
instead of creating a polyfill, you can just build it into the dist folder and use it from there, just like we did with the normal script. 

(for building I simply run `ncc build index.js`)